test/e2e/common/node/pod_level_resources.go

```diff
@@ -236,14 +236,19 @@ func verifyPodCgroups(ctx context.Context, f *framework.Framework, pod *v1.Pod,
 	cpuQuota := kubecm.MilliCPUToQuota(expectedResources.Limits.Cpu().MilliValue(), kubecm.QuotaPeriod)
 	expectedCPULimit := strconv.FormatInt(cpuQuota, 10)
 	expectedCPULimit = fmt.Sprintf("%s %s", expectedCPULimit, CPUPeriod)
-	err = e2epod.VerifyCgroupValue(f, pod, pod.Spec.Containers[0].Name, cpuLimCgPath, expectedCPULimit)
+
+	expectedCPULimitRoundedUp := strconv.FormatInt(e2epod.RoundUp(cpuQuota), 10)
+	expectedCPULimitRoundedUp = fmt.Sprintf("%s %s", expectedCPULimitRoundedUp, CPUPeriod)
+
+	err = e2epod.VerifyCgroupValue(f, pod, pod.Spec.Containers[0].Name, cpuLimCgPath, expectedCPULimit, expectedCPULimitRoundedUp)
 	if err != nil {
 		errs = append(errs, fmt.Errorf("failed to verify cpu limit cgroup value: %w", err))
 	}
 
 	memLimCgPath := fmt.Sprintf("%s/%s", podCgPath, cgroupv2MemLimit)
 	expectedMemLim := strconv.FormatInt(expectedResources.Limits.Memory().Value(), 10)
-	err = e2epod.VerifyCgroupValue(f, pod, pod.Spec.Containers[0].Name, memLimCgPath, expectedMemLim)
+	expectMemLimRoundedUp := strconv.FormatInt(e2epod.RoundUp(expectedResources.Limits.Memory().Value()), 10)
```

Comment:
ah actually just reread the issue, seems like we only need to worry about cpu limits, will update this to not round memory limits